runtimevar/httpvar/httpvar.go

```diff
@@ -36,6 +44,53 @@ import (
 	"time"
 )
 
+func init() {
+	o := &URLOpener{Client: http.DefaultClient}
+	for _, scheme := range Schemes {
+		runtimevar.DefaultURLMux().RegisterVariable(scheme, o)
+	}
+}
+
+// Schemes are the URL schemes httpvar registers its URLOpener under on runtimevar.DefaultMux.
+var Schemes = []string{"http", "https"}
+
+// URLOpener opens HTTP URLs like "http://myserver.com/foo.txt".
+//
+// The full URL, including scheme, is used as the endpoint, except that the
```

Comment:
How about just `the URL query values are removed`? Since we'll error out if there is unknown parameter.

Reply:
No, we don't... This one explicitly passes through all unknown query parameters.